src/components/TeamCard/TeamCard.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react"
+import "./TeamCard.scss"
+import LiLogo from "./fancylinkedin.svg"
+import cat from "./cat.jpg"
+
+interface CardProps {
+  name: string
+  imgsrc: string
+  title: string
+}
+
+const TeamCard: React.FC<CardProps> = ({ name, imgsrc, title }) => (
+  <div className='Card'>
+    <img src={imgsrc} alt={name} className='Card__Image' />
+    <p className='Card__Name'>{name}</p>
+    <p className='Card__Title'>{title}</p>
+    <div className='Card__Logo'>
+      <div className='Card__Logo__LogoBg'>
+        <img src={LiLogo} alt='LinkedIn Logo' />
+      </div>
```

Comment:
We are going to need to have a link to everyone's LinkedIn as a part of this image. Inside the google drive there is a contact list of all the organizers which should contain all the urls. You will probably need to change the interface as well to include that a LinkedIn Url.